source/simulator/solver.cc

```diff
@@ -190,9 +190,9 @@ namespace aspect
          * @param S_block_tolerance The tolerance for the CG solver which computes
```

Comment:
you should no longer pass in the S_block_tolerance here (and you need to update the comments here!) Make sure the S_inverse_operator is mentioned.